### DISCUSSION TITLE
User Statistics (Percentage)

### DISCUSSION THREAD

**Discussion by MannikJ** [spatie/flareapp.io-roadmap#56] [Ideas]
We monitor the errors of our B2B application with flare. We have some customers which are connected with their own systems via API. Often issues that occur are dependent on the customer, because their integrations are using the system differently from each other. Therefor, if unexpected errors occur, it's crucial to be able to inform the most affected users/customers early, so they can also work on their side to avoid further damage.  

To help with these kinds of errors which occur only or mainly for certain customers effectively and proactively, it would be very helpful if flare could display a percentage/share for each user.
I think of two things: 
1. Most affected users for each error
2. Most affected users over all errors

I'm not sure what would be visually the best. I could think about a pie chart or just a plain list with count for each user.

**Comment by metadeck** [comment]
We are also using a multi tenant approach to our application. Identifying which customers are having the most issues is paramount for us. +1 for the 2 metrics above.
